assets/js/blocks/featured-category/block.js

```diff
@@ -190,7 +190,7 @@ const FeaturedCategory = ( {
 
 		return (
 			<Placeholder
-				icon={ <Icon icon={ FolderStarred } /> }
+				icon={ <Icon srcElement={ FolderStarred } /> }
```

Comment:
I'm curious, why did you decide to export this as `FolderStarred` from the icon library instead of `folderStarred`? That seems inconsistent with using lowercase/camelCase for all other icons in the library?

Reply:
That’s definitely a late-night typo, it should be `folderStarred`, fixed.